### ISSUE TITLE
Missing form labels and fieldset

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#123]
The check boxes on the on the course admin assignment overview page are are missing form labels and are not enclosed in a fieldset.

**Comment by Levijatan** [comment]
Fixed in 4c3fda8 and c96a0b0
